charts/scaffold/README.md

```diff
@@ -1,52 +1,27 @@
 # scaffold
 
-![Version: 0.3.21](https://img.shields.io/badge/Version-0.3.21-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
+![Version: 0.3.25](https://img.shields.io/badge/Version-0.3.25-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
 
-Umbrella chart for deploying the components of the [sigstore](https://www.sigstore.dev/) architecture.
+Scaffolding the components of the sigstore architecture
 
-The following components are also included as either direct components or through chart dependencies:
+**Homepage:** <https://sigstore.dev/>
 
-* [ctlog](https://github.com/google/certificate-transparency-go)
-* [Fulcio](https://docs.sigstore.dev/fulcio/overview/)
-* [MySQL](https://www.mysql.com)
-* [Redis](https://redis.io)
-* [Rekor](https://docs.sigstore.dev/rekor/overview/)
-* [Trillian Log Server](https://github.com/google/trillian)
-* [Trillian Log Signer](https://github.com/google/trillian)
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| The Sigstore Authors |  |  |
 
 ## Requirements
 
 | Repository | Name | Version |
 |------------|------|---------|
-| https://sigstore.github.io/helm-charts | ctlog | 0.2.27 |
-| https://sigstore.github.io/helm-charts | fulcio | 0.4.12 |
-| https://sigstore.github.io/helm-charts | rekor | 0.3.21 |
+| https://sigstore.github.io/helm-charts | ctlog | 0.2.28 |
```

Comment:
it might be easier to update each chart in a separate PR, as you cant actually bump the dependency in different charts in the same PR.

README.md for scaffold picks up the new ctlog chart version 0.2.28, but the Chart.yaml still reference ctlog 0.2.27